### ISSUE TITLE
LB is not re-created when address is assigned after initial creation without address

### ISSUE THREAD

**Issue by RomeroGaliza** [terraform-google-modules/terraform-google-lb-http#370]
### TL;DR

When creating a LB without passing an address to it (therefore relying on the module to create the compute address), everything works fine, however, on a later stage, when adding a new address (external to the module) and setting `create_address` to `false` the LB seems to simply ignore this new input variable.

### Expected behavior

It seems like that once you create a LB, you cannot assign a new IP to it, the module simply ignores that input... a workaround was to comment out the the load balancer, apply, and then uncomment it, followed by one last apply.

### Terraform Configuration

```hcl
n/a
```

### Terraform Version

```sh
1.6.2
```

**Comment by imrannayer** [comment]
I dont see anything in module ignoring it.
